roles/bootstrap-os/tasks/bootstrap-centos.yml

```diff
@@ -78,6 +78,7 @@
     state: "{{ http_proxy | default(False) | ternary('present', 'absent') }}"
     no_extra_spaces: true
   become: true
+  when: proxy_yum_globally | bool
```

Comment:
How about just adding
```
  when:
    - http_proxy is defined
```
like the other Linux distribution cases instead of adding a new yum-specific option?

Reply:
That case would be taken care of by the default already in place on line 78.  In my use case, I need to define http_proxy for most of the environment. But, I manage the /etc/yum.conf and /etc/yum.repos.d separately due to an internally hosted Yum Proxy service I am required to use.  This variable allows me to turn off the yum proxy.  So that my preconfigured repo settings still work.  Without it, in my environment, Kubespray fails to download any of the preinstall packages because it is trying to reach my internal hosted yum repo using the proxy.

Reply:
Thanks for your explanation. I got an issue you are fixing.
I agree to add an option to skip using http_proxy for getting necessary OS packages.
One thing is I guess we will want to add other options when using different OS(like Ubuntu) because current option name is RHEL/CentOS specific.
How about more generic name like `skip_http_proxy_on_os_packages` or something?
It is fine to enable this option only for RHEL/CentOS only at this time if don't take care of the other OS distributions.

Reply:
I can do that